### DISCUSSION TITLE
Reverting to old Icon?

### DISCUSSION THREAD

**Discussion by artemsgithub** [VSCodium/vscodium#627] [General]
I missed it! Just wondering when that got done/decision making process.

**Comment by setanarut** [comment]
> I missed it! Just wondering when that got done/decision making process.

Which old icon are you talking about. the icon that looks like a green baby potty or the blue tree?

Related Issue and PR;
https://github.com/VSCodium/vscodium/issues/491
https://github.com/VSCodium/vscodium/issues/491#issuecomment-691597222
https://github.com/VSCodium/vscodium/pull/582

**Reply by artemsgithub** [reply]
> > I missed it! Just wondering when that got done/decision making process.
> 
> Which old icon are you talking about. the icon that looks like a green baby potty or the blue tree?
> 
> Related Issue and PR;
> #491
> [#491 (comment)](https://github.com/VSCodium/vscodium/issues/491#issuecomment-691597222)
> #582

The blue tree :) I didn't realize the other one was a baby pot, now I can't unsee. My friend said that it was reverted
